src/ssl/support.h

```diff
@@ -179,16 +179,24 @@ void unloadSquidUntrusted();
 void SSL_add_untrusted_cert(SSL *ssl, X509 *cert);
 
 /**
- * Searches in serverCertificates list for the cert issuer and if not found
- * and Authority Info Access of cert provides a URI return it.
+ * Retrieve the certificate issuer URI from Authority Info Access extension
+ * if this info exist.
+ \ingroup ServerProtocolSSLAPI
+ */
+const char *hasAuthorityInfoAccessCaIssuers(X509 *cert);
```

Comment:
We no longer use `\ingroup` labels in new code. Let's rename this function and simplify its description:

```C++
/// finds certificate issuer URI in the Authority Info Access extension
const char *findIssuerUriInAuthorityInfoAccess(X509*);
```

or, if there are no other places we might find the issuer URI in, then:

```C++
/// finds certificate issuer URI in the Authority Info Access extension
const char *findIssuerUri(X509*);
```

Reply:
commit 7372e0ebf